client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -139,6 +139,45 @@ export const getTotalPriceBreakdown = ( state, orderId, siteId = getSelectedSite
 		: null;
 };
 
+export const needsCustomsForm = createSelector(
```

Comment:
Does something like `isCustomsFormRequired` work better?

Reply:
Sure! I went ahead and changed it in the UI PR because it had a few new usaged of that function: https://github.com/Automattic/wp-calypso/pull/24974/commits/edf356ef2fdbe5ad4b39affe6604aeb1cf5d1342